src/login/LoginPage.jsx

```diff
@@ -195,7 +196,7 @@ class LoginPage extends React.Component {
               {this.props.loginError ? <LoginFailureMessage loginError={this.props.loginError} /> : null}
               {this.state.isSubmitted ? window.scrollTo({ left: 0, top: 0, behavior: 'smooth' }) : null}
               {activationMsgType && <AccountActivationMessage messageType={activationMsgType} />}
-              {this.props.forgotPassword.status === 'complete' ? <ConfirmationAlert email={this.props.forgotPassword.email} /> : null}
```

Comment:
Can we simply add another check in this condition like this?
```
{this.props.forgotPassword.status === 'complete' && !this.props.loginError ? <ConfirmationAlert email={this.props.forgotPassword.email} /> : null}
```

Reply:
yes. let me update the PR